apps/backend/src/services/configuration_monitoring.py

```
@@ -314,30 +463,30 @@ async def _get_discovered_config_paths(self, device: Device) -> List[str]:
+        discovered_paths: list[str] = []
+
         if not device.tags:
             return discovered_paths
-            
+
         try:
             # Get SWAG proxy config paths
             if device.tags.get("swag") and device.tags.get("swag_running"):
                 # Look for SWAG config paths in analysis results
                 analysis = device.tags.get("last_analysis", {})
                 services = analysis.get("services", {}) if isinstance(analysis, dict) else {}
```

Ruff (0.12.2)

Local variable `services` is assigned to but never used

Remove assignment to unused variable `services`

(F841)